chainforge/react-server/src/backend/utils.ts

```diff
@@ -782,14 +782,56 @@ export async function call_alephalpha(prompt: string, model: LLM, n: number = 1,
   return [query, responses];
 }
 
+export async function call_ollama_provider(prompt: string, model: LLM, n: number = 1, temperature: number = 1.0, params?: Dict): Promise<[Dict, Dict]> {
+  const url: string = params.ollama_url;
+
+  // FIXME: Ollama doesn't support batch inference, but llama.cpp does so it will eventually
```

Comment:
This limitation is a bit unfortunate. However, llama.cpp does support batch inference so maybe ollama will add support for it soon.